backend/services/roles.js

```diff
@@ -0,0 +1,9 @@
+const { Role } = require("../models");
+
```

Comment:
Can we add a comment for this function just for consistency?

Reply:
done!